internal/executor/instance/persistentworker/isolation/none/none.go

```diff
@@ -23,9 +24,19 @@ type PersistentWorkerInstance struct {
 	cleanup func() error
 }
 
+func staticTempDirWithDynamicFallback() (string, error) {
+	// Prefer static directory for non-Cirrus CI caches efficiency (e.g. ccache)
+	staticTempDir := filepath.Join(os.TempDir(), "cirrus-cli-build")
```

Comment:
Let's do `cirrus-build` and maybe we'll change it in the cloud too for consistency

```suggestion
	staticTempDir := filepath.Join(os.TempDir(), "cirrus-build")
```

Reply:
Why? The `ci`/`cli` designation might aid in debugging.

Reply:
To possibly reuse caches between worker tasks and regular container tasks (in case caches contain absolute paths). 🤷‍♂️

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/360/commits/2ccc7fa4f1df281eb56272143d0825a06dc21d7c.